main.nf

```diff
@@ -21,78 +21,66 @@ version = '0.1'
 
 // Configurable variables - same as NGI-RNAseq for now
 params.project = false
-params.genome = false
-params.forward_stranded = false
-params.reverse_stranded = false
-params.unstranded = false
-params.star_index = params.genome ? params.genomes[ params.genome ].star ?: false : false
-params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
-params.gtf = params.genome ? params.genomes[ params.genome ].gtf ?: false : false
-params.bed12 = params.genome ? params.genomes[ params.genome ].bed12 ?: false : false
-params.hisat2_index = params.genome ? params.genomes[ params.genome ].hisat2 ?: false : false
-params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
-params.splicesites = false
-params.download_hisat2index = false
-params.download_fasta = false
-params.download_gtf = false
-params.hisatBuildMemory = 200 // Required amount of memory in GB to build HISAT2 index with splice sites
-params.saveReference = false
-params.saveTrimmed = false
-params.saveAlignedIntermediates = false
 params.reads = "data/*{1,2}.fastq.gz"
 params.outdir = './results'
 params.email = false
-params.STAR = false
-params.FUSIONCATCHER = true
+params.star = false
+params.fusioncatcher = true
 Channel
-    .fromFilePairs( params.reads, size:  2 )
+    .fromFilePairs( params.reads, size: 2 )
     .ifEmpty { exit 1, "Cannot find any reads matching: ${params.reads}" }
-    .into { read_files_STAR-fusion; fusionInspector-reads; fusioncatcer-reads}
-
+    .into { read_files_star_fusion; fusion_inspector_reads; fusioncatcher_reads}
 
 
 /*
- * STEP 1 - STAR-Fusion 
+ * STEP 1 - STAR-Fusion
  */
+process star_fusion{
 
-process STAR_fusion{
-        
     input:
-    set val (name), file(read1), file(read2) from read_files_STAR-fusion   
+    set val (name), file(read1), file(read2) from read_files_star_fusion
 
     output:
-    '*final.abridged*'
-    'star-fusion.fusion_candidates.final.abridged.FFPM' into fusion_candidates
-    
-    when: STAR == true 
```

Comment:
I don't think so, I was looking at CAW when I wrote this: https://github.com/SciLifeLab/CAW/blob/master/main.nf#L244
Maybe I missunderstood something